server/server.py

```diff
@@ -186,42 +259,31 @@ def uploadSticker():
     else:
         return json.dumps({'status': '405', 'error': 'HTTP Method not allowed.'}), 405
 
-# @app.route('/addEmail', methods=['PATCH'])
-# def addEmail():
-#     # Check token if required
-#     if os.getenv('STICKER_MAP_REQUIRE_LOGIN') == "True":
-#         if not checkToken(request.cookies.get('token')):
-#             return json.dumps({'status': '403', 'error': 'Not authenticated or cookies disabled.'}), 405
-#     print(f"{request.form["email"]}, {request.form["token"]}")
-#     if request.form['email'] != '':
-#         if request.form['token'] != '':
-#             # Check if the token is in the database
-#             with Session(engine) as session:
-#                 stmt = select(Sticker).where(Sticker.adderemail.is_(request.form['token']))
-#                 sticker = session.query(Sticker).from_statement(stmt).one_or_none()
-#                 print(sticker)
-#                 if sticker is not None:
-#                     # Change email in database
-#                     sticker.adderemail = request.form["email"]
-#                     session.commit()
+@app.route('/addEmail', methods=['PATCH'])
+def addEmail():
```

Comment:
What is the email added to? The function is quite vague, only upon reading the code it is evident that you can add emails to a sticker.